infra/aggregate.py

```diff
@@ -0,0 +1,18 @@
+from sys import stdout
+from os import listdir
+from os.path import join
+import json
+
+root="./tmp/bench"
```

Comment:
ah, sweet sweet python
Please wrap the whole script in a function and call it at the bottom of the file for python reasons

Reply:
Sure. Do you want me to do the whole `if __name__ == 'main'`?

Reply:
Nah just a function is good